### ISSUE TITLE
CMB2 issue with hook cmb2_admin_init

### ISSUE THREAD

**Issue by samikeijonen** [WordImpress/google-maps-builder#129]
I got this kind of error when I try to switch hook `cmb2_init` to `cmb2_admin_init` as current version is [suggesting](https://github.com/WebDevStudios/CMB2/blob/master/example-functions.php).

```
Fatal error: Cannot redeclare cmb2_dir() (previously declared in ...\wp-content\plugins\cmb2\includes\helper-functions.php:18) in ...\wp-content\plugins\google-maps-builder\includes\libraries\metabox\includes\helper-functions.php on line 20
```

**Comment by DevinWalker** [comment]
Hey there - Can you please provide more context as to what you're trying to achieve? I see you have the CMB2 Plugin installed alongside Maps Builder - do you wish to run the two simultaneously?

**Comment by samikeijonen** [comment]
Yes I have CMB2 plugin installed. I didn't realize that you have bundled it in your plugin also. Should there be some kind of check that if the plugin is around.

After that it's just basic metabox settings for custom theme.

**Comment by DevinWalker** [comment]
Yes I've seen more themes including CMB2 as a plugin and I have the code already to add to Maps Builder to check for it installed. I'll make this a priority. Thanks for reporting.

**Comment by samikeijonen** [comment]
Thanks. I personally use [plugin version](https://wordpress.org/plugins/cmb2/) because in that way I don't have to keep track of the changes and update my custom theme/plugin.

And I pretty often register metaboxes in custom plugin also.
